src/components/views/dialogs/UserSettingsDialog.js

```diff
@@ -110,6 +112,14 @@ export default class UserSettingsDialog extends React.Component {
             <PreferencesUserSettingsTab />,
         ));
 
+        if (SettingsStore.getValue("feature_keybindings")) {
```

Comment:
When toggling this lab setting you have to close and reopen the modal to the the keybindings tab.
It would probably be appropriate to watch that settings the same way it is done for `feature_mjolnir` in `componentDidMount`

Reply:
I didn't do this on purpose. The `TabbedView` uses a number to keep the track of the currently opened tab. That means that when I add a new tab before the current one it will switch to the tab before the currently selected tab.